oci/archive/oci_transport.go

```diff
@@ -183,7 +183,12 @@ func createUntarTempDir(sys *types.SystemContext, ref ociArchiveReference) (temp
 	src := ref.resolvedFile
 	dst := tempDirRef.tempDirectory
 	// TODO: This can take quite some time, and should ideally be cancellable using a context.Context.
-	if err := archive.UntarPath(src, dst); err != nil {
+	archiver, err := os.Open(src)
```

Comment:
```suggestion
	archive, err := os.Open(src)
```

Reply:
use arch instead of archive since there's name confliction with package storage/pkg/archive